# Array of 3D objects
Header header
Object3D[] objects
